app/models/goal.py

```diff
@@ -2,4 +2,19 @@
 
 
 class Goal(db.Model):
```

Comment:
Remember to make an empty `__init__.py` file in any package folder/subfolder. app has one, but we should have one here in the models folder as well.